version.php

```diff
@@ -24,7 +24,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2020040300; // The current plugin version (Date: YYYYMMDDXX).
+$plugin->version   = 2024091300; // The current plugin version (Date: YYYYMMDDXX).
```

Comment:
I would suggest to bump it as minimal as possible to avoid any possible clashes with the plugin maintainer changes. Sometimes they do versions differently to what we do and etc. Please change it to 2020040301